### DISCUSSION TITLE
Multi-tenancy play 1.7.1

### DISCUSSION THREAD

**Discussion by ikk1** [playframework/playframework#11390] [General]
Hi 😊!
 I’ve already a play 1 application that runs with a single database. I search a lot to find any tutorial / documentation that explain how to do that.

Is there a way to achieve that? I think that this will be achieve by multiple schemas and the correct schema to connect will be defined by url. 

Please someone guide me through :pray:

**Comment by mkurz** [comment]
Since this question is about Play 1 you should ask over at their repository: https://github.com/playframework/play1 This repos here (and actually all within the Play Framework GitHub organization, except for play1, are about Play 2.

About your request: There are many ways to build multi tenant applications. It's a general question, not really specific to any web framework. Depending on your needs, partition by schema might work, however I recommend to look into row level security within PostgreSQL, where your tenant share the same schema and table(s). There are various discussions about that, e.g. see https://news.ycombinator.com/item?id=32241820

**Reply by ikk1** [reply]
Thanks for your reply @mkurz, i'll follow your orientation.
